SoftLayer/CLI/block/subnets/assign.py

```diff
@@ -0,0 +1,31 @@
+"""Assign block storage subnets to the given host id."""
+# :license: MIT, see LICENSE for more details.
+
+import click
+import SoftLayer
+from SoftLayer.CLI import environment
+
+
+@click.command()
+@click.argument('access_id')
```

Comment:
what is an `access_id`? How would a customer get that?

You also say `host id` in the description, but use `access_id` here, so I'm not sure if those are the same or what.

Reply:
access_id is indeed the host_id. I am using it to maintain coding style, since the slcli is using access-list in the command, and other functions in manager/block.py use access_id  as well.

I can change to host_id if desired.